vowpalwabbit/OjaNewton.cc

```diff
@@ -533,14 +526,14 @@ base_learner* OjaNewton_setup(arguments& arg)
 {
   auto ON = scoped_calloc_or_throw<OjaNewton>();
   if (arg.new_options("OjaNewton options")
-      .critical("OjaNewton", "Online Newton with Oja's Sketch")
-      ("sketch_size", ON->m, 10, "size of sketch")
-      ("epoch_size", ON->epoch_size, 1, "size of epoch")
-      ("alpha", ON->alpha, 1.f, "mutiplicative constant for indentiy")
-      ("alpha_inverse", po::value<float>(), "one over alpha, similar to learning rate")
-      ("learning_rate_cnt", ON->learning_rate_cnt, 2.f, "constant for the learning rate 1/t")
-      ("normalize", ON->normalize, true, "normalize the features or not")
-      ("random_init", ON->random_init, true, "randomize initialization of Oja or not").missing())
+          .critical("OjaNewton", "Online Newton with Oja's Sketch")("sketch_size", ON->m, 10, "size of sketch")(
```

Comment:
Can we change an option to not cause this change?

Reply:
I haven't come across any options that I think would be relevant. Would adding comments to turn off `clang-format` around blocks that are roughly "tabular" be acceptable?

Reply:
What's involved with turning it off?